src/config.js

```diff
@@ -6,10 +6,10 @@ const NOMOS_BASE_URI = process.env.NOMOS_BASE_URI || "https://membership.vanhack
 
 function resolveApiKey() {
   if (process.env.NOMOS_API_KEY_FILE) {
-    return fs.readFileSync(process.env.NOMOS_API_KEY_FILE);
+    return fs.readFileSync(process.env.NOMOS_API_KEY_FILE).trim();
   }
 
-  return process.env.NOMOS_API_KEY || "";
+  return process.env.NOMOS_API_KEY.trim() || "";
```

Comment:
this should be `(process.env.NOMOS_API_KEY).trim()` otherwise if the env is undefined/null this will throw

Reply:
I mean `(process.env.NOMOS_API_KEY || "").trim()`

Reply:
^ this

Reply:
.